File: client/src/app/_models/orders/orderForwardToAgent.ts
import { IOrderForwardCategory } from "./orderForwardCategory";

export interface IOrderForwardToAgent
{
	id: number;
	orderId: number;
	orderNo: number;
	orderDate: Date;
	customerId: number;
	customerName: string;
	customerCity: string;
	projectManagerId: number;
	orderForwardCategories: IOrderForwardCategory[];
}

export class OrderForwardToAgent 
{
	id= 0;
	orderId= 0;
	orderNo= 0;
	orderDate: Date = new Date('1900-01-01');
	customerId= 0;
	customerName= '';
	customerCity= '';
	projectManagerId= 0;
	orderForwardCategories: IOrderForwardCategory[]=[];
}

File: api/Entities/Admin/Order/OrderForwardToAgent.cs
namespace api.Entities.Admin.Order
{
    public class OrderForwardToAgent: BaseEntity
    {
        public int OrderId {get; set;}
        public int OrderNo {get; set;}
        public DateTime OrderDate {get; set;}
        public int CustomerId {get; set;}
        public string customerName {get; set;}
        public string CustomerCity {get; set;}
        public int ProjectManagerId {get; set;}
        //public ICollection<OrderForwardCategory> OrderForwardCategories {get; set;}
    }
}